server/functions/fileSystemFunctions.js

```diff
@@ -247,6 +280,23 @@ async function addFunctionString(applicationId, filename, code) {
     await waitForNewFilesToBeRead()
 }
 
+async function addDependencies(applicationId, newDependencies, isDev) {
+    const packageString = `${newDependencies}${isDev ? '--dev' : ''}`;
+    
+    const appDirectory = `${config.functionsRoot}${applicationId}`;
```

Comment:
Is `config.functionsRoot` expected to have a trailing "/"?

Reply:
`Path.join` might be the move if it's not

Reply:
Agree, in this case I was keeping with the style of how the rest of the file was written